posix.mak

```diff
@@ -54,8 +54,6 @@ ifneq ($(BUILD),release)
     endif
 endif
 
-override PIC:=$(if $(PIC),-fPIC,)
-
```

Comment:
What about this instead? Would at least allow to still build phobos without PIC, might be interesting for some use-cases.
```make
ifneq (0,$(PIC))
    DEFAULT_PIC:=-fPIC
else
    DEFAULT_PIC:=
endif
```